sportsdataverse/cfb/cfb_pbp.py

```
@@ -2937,11 +3011,12 @@ def __setup_penalty_data(self, play_df):
                 .otherwise(None),
             )
             .with_columns(
-                # Direct signed-integer capture. The prior fragment capture
-                # (".{0,3} yards") produced garbage on 40% of rows measured
-                # against 2004-2025 ('U (', 'k 8'); a numeric grab cannot.
                 yds_penalty=pl.when(pl.col("penalty_flag") == True)
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)